Server/JokesOnYou.Web.Api/Controllers/AccountController.cs

```diff
@@ -23,25 +23,25 @@ namespace JokesOnYou.Web.Api.Controllers
     [ApiController]
     public class AccountController : ControllerBase
     {
-        private readonly IUserService _userService; 
+        private readonly IUserService _userService;
 
         public AccountController(IUserService userService)
-        { 
+        {
             _userService = userService;
         }
 
         [HttpPost("login")]
-        public async Task<ActionResult> Login (UserLoginDTO userLogin)
+        public async Task<ActionResult<UserReplyDTO>> Login (UserLoginDTO userLogin)
         {
-            var loginResult = await _userService.LoginUser(userLogin);
-            return Ok(loginResult); 
+            var userReplyDTO = await _userService.LoginUser(userLogin);
+            return Ok(userReplyDTO);
```

Comment:
Since you specified the return type of the method it should be ok to skip the Ok() wrapper in order to provide type safety
- side note: doesnt work with ienumerables.

Reply:
Oooh that's interesting now i see it, seems like last time i actually didn't get it ;). now i understand, thanks!

Reply:
removed wrapper